free-programming-books-es.md

```diff
@@ -351,6 +352,7 @@
 ### Scratch
 
 * [Informática Creativa](https://github.com/programamos/GuiaScratch) (PDF)
+* [Manual de Scratch 2](https://lsi.vc.ehu.es/pablogn/docencia/FdI/Scratch/Aprenda%20a%20programar%20con%20Scratch%20en%20un%20par%20de%20tardes.pdf) (PDF)
```

Comment:
Good resource, but the site does not have a valid certificate for https. Use http instead.